### ISSUE TITLE
Missing function calls

### ISSUE THREAD

**Issue by rfalke** [BoomerangDecompiler/boomerang#123]
Version: ed7a3e1
Exe: https://github.com/rfalke/decompiler-subjects/blob/master/from_holdec/dmi/output/ia32_elf/subject.exe
Source:
```
int main(int argc, char *argv[])
{
        int result = 0;
        result += basic_1_ternary_simple(argc);
        result += basic_2_ternary_other(argc);
        result += intermediate_1_ternary_blocks(argc);
        result += intermediate_2_other_in_blocks(argc);
        result += intermediate_10_double_word_signed_math(argc, argc, argc);
        result += intermediate_10_double_word_unsigned_math(argc, argc, argc);
        result += advanced_1_unrolled_loop();

        return result + argv[0][0];
}
```

Output:

```
/** address: 0x08048340 */
int main(int argc, char *argv[])
{
    __size8 al;                 // r8
    __size32 eax;               // r24
    __size32 ebp;               // r29
    __size32 ebx;               // r27
    int edi;            // r31
    int esi;            // r30
    union { __size32; __size32 *; } esp;                // r28
    __size32 local4;            // m[esp]

    eax = basic_1_ternary_simple(argc); /* Warning: also results in al */
    basic_2_ternary_other(al, eax, esp + 4, argv, eax, esp - 8, argc, argc >> 31, SARFLAGS(argc >> 31, argc, 31), SARFLAGS(argc >> 31, argc, 31), SARFLAGS(argc >> 31, argc, 31), a
rgc, argv, local4, ebp, edi, esi, ebx, esp + 4, argv, argc, pc);
}
```

* The other function calls are not there
* There is no return statement for a non-void function
* The result of the basic_2_ternary_other isn't used
* minor: the parameters of basic_2_ternary_other are wrong

**Comment by ceeac** [comment]
This is caused by the same issue as in #122 (missing `cmovs` semantics).

**Comment by ceeac** [comment]
This is fixed now that #69 is merged.
